reference/7.2/Microsoft.PowerShell.Core/About/about_Comparison_Operators.md

```diff
@@ -377,12 +377,30 @@ Collection examples:
 #Output: Bag, Beg
 ```
 
-`-match` and `-notmatch` support regex capture groups. Each time they run, they
-overwrite the `$Matches` automatic variable. When `<input>` is a collection the
```

Comment:
@chasewilson the current text in this paragraph is wrong and either needs to be corrected or removed.  I think it is useful to have the corrected text on when `$Matches` is set in this topic.  Partially because this is where I'd want to see the examples on guarding access to `$Matches` when using `-match/-notmatch`.

Reply:
I can agree with that. I'm going to merge in my PR as it's relevant to that doc. If you can merge in those changes that would avoid any merge conflicts. I'll take a look at your wording as well to see if I can help. 

@kurtmckee, that's a great catch I'll review to ensure formatting. FYI variables, including booleans, as they are written in the terminal are code fenced. e.g. `$true`

Reply:
OK, I've rebased onto the latest changes on `staging`.

Reply:
While your new information is true, I think the information about the capture groups is still relevant. It looks to me that these are two separate sections of information.  Can you break these out?

Reply:
Sure. I'll do that tonight.

Reply:
@chasewilson Have you had a chance to see if the requested changes look good?